### PR TITLE
Fix audit errors and light styling

### PR DESCRIPTION
**Pull request by bhstahl**
- Added sizes & alt tags to images
- Hover state transitions on links
- Light styling and spacing to Tiny Letter form
- Overall cleanup for readability

### CODE REVIEW

**Comment by alexlines** [comment]
Awesome, thanks @bhstahl! There are some much-needed cleanups here which were on my list. In general I prefer isolating whitespace changes in their own commits as well as breaking commits into concept-commits, e.g. "move google analytics to the page's last script tag" etc ... but I'm nitpicking because I can't help myself.

Huge thanks dude, @nchirls and I owe you many :beers:!

File: index.html [hunk 1]
```diff
@@ -16,8 +15,11 @@
     <meta name="twitter:description" content="Notation Capital is a venture capital fund in New York.">
     <meta name="twitter:image:src" content="http://notationcapital.com/assets/images/notation-capital-black-horizontal-@2x.png">
 
+    <!-- Fonts -->
```
Comment (alexlines) [inline index.html:18]: nice commenting, thx

File: index.html [hunk 2]
```diff
@@ -27,72 +29,67 @@
...
+    </div>
     <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
     <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> -->
-
-  <script src="assets/js/cowsay.min.js"></script>
-  <script> console.cowsay("Notation"); </script>
-
+    <script src="assets/js/cowsay.min.js"></script>
+    <script> console.cowsay("Notation"); </script>
+    <script>
```
Comment (alexlines) [inline index.html:83]: yes, classic chartbeat learnings :+1:

File: index.html [hunk 3]
```diff
@@ -27,72 +29,67 @@
...
+          <p>Established 2015 NYC</p>
+      </footer>
+    </div>
     <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
     <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> -->
-
-  <script src="assets/js/cowsay.min.js"></script>
-  <script> console.cowsay("Notation"); </script>
-
+    <script src="assets/js/cowsay.min.js"></script>
```
Comment (alexlines) [inline index.html:81]: shouts to @jeffreytierney :cow2:

File: assets/css/notation.css [hunk 4]
```diff
@@ -67,36 +72,152 @@ a:hover, a:active {
...
+    -ms-transition: all 150ms ease-in;
+    transition: all 150ms ease-in;
+    -moz-border-radius: 1px;
+    -webkit-border-radius: 1px;
+    border-radius: 1px;
+}
+
+/* INPUT -------------------*/
+
+.tinyletter-input {
```
Comment (alexlines) [inline assets/css/notation.css:140]: :+1: on tinyletter styling

File: assets/css/notation.css [hunk 5]
```diff
@@ -1,64 +1,69 @@
 html {
-        height: 100%;
+    height: 100%;
 }
 
 body {
-        background-color: #000;
```
Comment (alexlines) [inline assets/css/notation.css:6]: mixing in whitespace changes :see_no_evil:

File: index.html [hunk 6]
```diff
@@ -27,72 +29,67 @@
...
+            <img class="social-img" src="assets/images/instagram-48.png" alt="Notation Capital - Instagram">
+          </a>
+          <a href="https://twitter.com/NotationCapital/">
+            <img class="social-img" src="assets/images/twitter-48.png" alt="Notation Capital - Twitter">
+          </a>
+          <a href="mailto:connect@notationcapital.com">
+            <img class="social-img" src="assets/images/email-48.png" alt="Notation Capital - Email">
+          </a>
+          <a href="https://goo.gl/maps/KXrTw">
+            <img class="social-img" src="assets/images/ic_place_white_18dp.png" alt="Notation Capital - Map">
```
Comment (alexlines) [inline index.html:63]: map pin is actually 36x36 so this blows it up a little too large 
icon_comp

File: assets/css/notation.css [hunk 7]
```diff
@@ -67,36 +72,152 @@ a:hover, a:active {
...
+a:hover .social-img{
+    -ms-transform: translateY(-3px);
+    -webkit-transform: translateY(-3px);
+    transform: translateY(-3px);
+}
+
+.social-img {
+    width: 48px;
+    height: 48px;
+    -webkit-transition: all 100ms ease-out;
```
Comment (alexlines) [inline assets/css/notation.css:88]: :+1:

File: assets/css/notation.css [hunk 8]
```diff
@@ -67,36 +72,152 @@ a:hover, a:active {
...
+}
+
+input::-webkit-input-placeholder {
+    color: #B18410;
+}
+input:focus::-webkit-input-placeholder {
+    color: #D0D0D0;
+}
+
+/* Firefox < 19 */
```
Comment (alexlines) [inline assets/css/notation.css:165]: :+1:

**Comment by bhstahl** [comment]
@alexlines I know...I just started tinkering and then all of a sudden realized I had made a ton of mixed changes and hit a stalemate. Thought about wiping and starting over to split up but said fuggit. Totally happy to do that for sanity sake though

File: assets/css/notation.css [hunk 9]
```diff
@@ -67,36 +72,152 @@ a:hover, a:active {
...
 
-@media 
-(-webkit-min-device-pixel-ratio: 2), 
-(min-resolution: 192dpi) { 
-        /*
+@media
+(-webkit-min-device-pixel-ratio: 2),
+(min-resolution: 192dpi) {
+/*
    h1 {
```
Comment (alexlines) [inline assets/css/notation.css:215]: more debt (of mine) here ...

File: index.html [hunk 10]
```diff
@@ -27,72 +29,67 @@
...
+        <h2 class="col-xs-12 col-sm-12">Connect</h2>
         <p class="col-xs-12 col-sm-12">
-          <a href="https://github.com/notationcapital"><img src="assets/images/github-48.png"></a>
-          <a href="https://angel.co/notation-capital"><img src="assets/images/angellist-48.png"></a>
-          <a href="https://instagram.com/NotationCapital/"><img src="assets/images/instagram-48.png"></a>
-          <a href="https://twitter.com/NotationCapital/"><img src="assets/images/twitter-48.png"></a>
-          <a href="mailto:connect@notationcapital.com"><img src="assets/images/email-48.png"></a>
-          <a href="https://goo.gl/maps/KXrTw"><img src="assets/images/ic_place_white_18dp.png"></a>
+          <a href="https://github.com/notationcapital">
+            <img class="social-img" src="assets/images/github-48.png" alt="Notation Capital - Github">
```
Comment (alexlines) [inline index.html:48]: :+1: on adding alt tags

**Comment by alexlines** [comment]
@bhstahl totally not necessary, the work is good and much appreciated. I'm just contractually obligated to mention these points :neckbeard:

**Comment by alexlines** [comment]
I'll probably fix the map icon size next, but good for now. merged & pushed. :dancer: :beers:

**Comment by bhstahl** [comment]
Ah I should have told you to hold! I was about to split things up because I'm ocd. Fixing the map icon though, not sure how I missed that resize.

**Comment by alexlines** [comment]
all good dude, don't need to create more work for you, your free :beer: counter is pushing double digits already

**Pull request merged**